api/v1/backupstoragelocation_types.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 the Velero contributors.
+Copyright 2018, 2020 the Velero contributors.
```

Comment:
Thinking through the implications of our API types living in two separate packages, and also if we need to consider this a breaking change. It will be breaking for anyone who vendors our code to use the API/client. That said, we haven't historically actually maintained backwards compatibility in our exported API surface..